include/SugarObjects/templates/basic/Basic.php

```diff
@@ -82,76 +88,100 @@ public function get_summary_text()
      * @throws RuntimeException
      * @throws InvalidArgumentException
      */
-    public function getEmailAddressConfirmOptInTick($emailField)
+    public function getConfirmOptInTickFromSugarEmailAddressField($emailField)
     {
-        $this->getEmailAddressValidateArguments($emailField);
+        $this->validateSugarEmailAddressField($emailField);
 
-        global $sugar_config, $app_strings;
+        return $this->displayOptInFromSugarEmailAddressField($emailField);
+    }
 
-        $tickHtml = '';
-        $confirmOptIn = $this->getEmailAddressConfirmOptIn($emailField);
-        if ($confirmOptIn && $sugar_config['email_enable_confirm_opt_in']) {
-            $tickTitle = $app_strings['LBL_CONFIRM_OPT_IN_TITLE'];
-            $tickHtml = '<span class="confirm-opt-in-tick" title="' . $tickTitle . '">&#10004;</span>';
+
+    /**
+     * @global array $sugar_config
+     * @global \LoggerManager $log
+     * @param string $emailField
+     * @return string
+     */
+    public function getOptInStatusFromSugarField($emailField)
```

Comment:
Update php doc: This method doesn't use globals (`$sugar_config` and `$log`) anymore.

Reply:
I have removed them.